draft-ietf-quic-http.md

```diff
@@ -623,31 +621,28 @@ allows the server push to be associated with a client request.  Promised
 requests MUST conform to the requirements in Section 8.2 of {{!HTTP2}}.
 
 The same server push can be associated with additional client requests using a
-DUPLICATE_PUSH frame (see {{frame-duplicate-push}}).
+PUSH_PROMISE frame with the same push id on multiple request stream.
 
-Ordering of a PUSH_PROMISE or DUPLICATE_PUSH in relation to certain parts of the
-response is important. The server SHOULD send PUSH_PROMISE or DUPLICATE_PUSH
-frames prior to sending HEADERS or DATA frames that reference the promised
-responses.  This reduces the chance that a client requests a resource that will
-be pushed by the server.
+Ordering of a PUSH_PROMISE in relation to certain parts of the response is
+important. The server SHOULD send PUSH_PROMISE frames prior to sending HEADERS
+or DATA frames that reference the promised responses.  This reduces the chance
+that a client requests a resource that will be pushed by the server.
 
 When a server later fulfills a promise, the server push response is conveyed on
 a push stream (see {{push-streams}}). The push stream identifies the Push ID of
 the promise that it fulfills, then contains a response to the promised request
 using the same format described for responses in {{request-response}}.
 
-Due to reordering, DUPLICATE_PUSH frames or push stream data can arrive before
-the corresponding PUSH_PROMISE frame.  When a client receives a DUPLICATE_PUSH
-frame for an as-yet-unknown Push ID, the request headers of the push are not
-immediately available.  The client can either delay generating new requests for
-content referenced following the DUPLICATE_PUSH frame until the request headers
-become available, or can initiate requests for discovered resources and cancel
-the requests if the requested resource is already being pushed. When a client
-receives a new push stream with an as-yet-unknown Push ID, both the associated
-client request and the pushed request headers are unknown.  The client can
-buffer the stream data in expectation of the matching PUSH_PROMISE. The client
-can use stream flow control (see section 4.1 of {{QUIC-TRANSPORT}}) to limit the
-amount of data a server may commit to the pushed stream.
+Due to reordering, push stream data can arrive before the corresponding
+PUSH_PROMISE frame.  When a client receives a new push stream with an
+as-yet-unknown Push ID, both the associated client request and the pushed
+request headers are unknown.  The client can buffer the stream data in
+expectation of the matching PUSH_PROMISE. The client can use stream flow control
+(see section 4.1 of {{QUIC-TRANSPORT}}) to limit the amount of data a server may
+commit to the pushed stream.
+
+If the same Push ID is promised on multiple request streams, the decompressed
```

Comment:
I might be inclined to put this at the end of the paragraph on lines 623/624?

Reply:
I think we can remove this sentence now that it is incorporated into the paragraph starting at line 605.

Sorry for me not being explicit about the move when suggesting changes to that paragraph.

Reply:
agree with Kazuho but also that the the sentence should use Mike's suggestion of comparison rather than "byte-for-byte"

Reply:
Done.